From 1bc2c8ad62c4e0c539ca43413fb403ef672bf817 Mon Sep 17 00:00:00 2001
Message-ID: <1bc2c8ad62c4e0c539ca43413fb403ef672bf817.1791896719.git.marcel.ziswiler@codethink.co.uk>
In-Reply-To: <f458a1bc3fdfbc3c3a17b53414a6c88bbf7e7733.1791896719.git.marcel.ziswiler@codethink.co.uk>
References: <f458a1bc3fdfbc3c3a17b53414a6c88bbf7e7733.1791896719.git.marcel.ziswiler@codethink.co.uk>
From: Min Lin <linmin@eswincomputing.com>
Date: Wed, 14 Jan 2026 13:13:08 +0800
Subject: [PATCH 29/31] riscv: dts: eswin:set 1.4GHz as the maximum cpu
 frequency
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Modified eic7700-opptable.dtsi to set 1.4GHz as the maximu cpu frequency.
To run the CPU above 1.4GHz requires increasing the voltage, but the driver
for that voltage boost hasn’t been merged yet.

Signed-off-by: Min Lin <linmin@eswincomputing.com>

Upstream-Status: Pending
---
 arch/riscv/boot/dts/eswin/eic7700-opptable.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/riscv/boot/dts/eswin/eic7700-opptable.dtsi b/arch/riscv/boot/dts/eswin/eic7700-opptable.dtsi
index 5d55a419ef64..d065dcdf1be1 100644
--- a/arch/riscv/boot/dts/eswin/eic7700-opptable.dtsi
+++ b/arch/riscv/boot/dts/eswin/eic7700-opptable.dtsi
@@ -80,6 +80,7 @@ opp-1400000000 {
 			opp-microvolt = <800000>;
 			clock-latency-ns = <70000>;
 		};
+		/*
 		opp-1500000000 {
 			opp-hz = /bits/ 64 <1500000000>;
 			opp-microvolt = <900000>;
@@ -100,6 +101,7 @@ opp-1800000000 {
 			opp-microvolt = <900000>;
 			clock-latency-ns = <70000>;
 		};
+		*/
 	};
 
 	dsp_opp_table: opp-table@dsp {
-- 
2.53.0

